static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -286,20 +291,20 @@ function DropdownItem({
   onClick,
   onIconClick,
   additionalSearchConfig,
+  customInvalidTagMessage,
 }: DropdownItemProps) {
   const isDisabled = item.value === null;
 
   let children: React.ReactNode;
   if (item.type === ItemType.RECENT_SEARCH) {
     children = <QueryItem item={item} additionalSearchConfig={additionalSearchConfig} />;
   } else if (item.type === ItemType.INVALID_TAG) {
-    children = (
-      <Invalid>
-        {tct("The field [field] isn't supported here. ", {
+    children = customInvalidTagMessage?.(item) ?? (
+      <SearchInvalidTag
+        message={tct("The field [field] isn't supported here. ", {
```

Comment:
Should their be a trailing space here?
```suggestion
        message={tct("The field [field] isn't supported here.", {
```